metadata/release-index.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
should we include stream name in release-index.json as well? Can be useful to figure out stream while looking at the json file.

Reply:
The release-index is inside of a stream specific location (e.x.: `s3://fcos-builds/prod/streams/<stream>/release-index.json`)

Reply:
Followup: the release-index is also specific to an architecture. Thus, architecture label should be part of the location, or somewhere in the index.

Reply:
@lucab: Reading the code again with more coffee leads me to believe that I misread something earlier and that is not the case.

Each entry in `release-index` points to an endpoint for the `release.json` which IS stream specific but is NOT architecture specific.

e.x.:

`s3://fcos-builds/prod/streams/foo/release-index.json` is the release-index for the foo stream. An example entry would be something like:
```json
{
    "commit": "<hash>",
    "version": "30.123.45",
    "endpoint": "https://fcos-builds.s3.amazonaws.com/prod/streams/foo/builds/30.123.45/release.json"
}
```